code/game/objects/structures/vehicles/carts/mining_cart.dm

```diff
@@ -3,7 +3,7 @@
 	icon = 'icons/obj/vehicles.dmi'
 	icon_state = "mining_cart"
 
-/*/obj/cart/mining_cart/Move()
+/*/obj/cart/mining_cart/Move(NewLoc, Dir = 0, step_x = 0, step_y = 0, glide_size_override = 0)
```

Comment:
lol

Reply:
What even

Reply:
They must have used the CTRL H replace all function